README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to [localhost:8080/]()
+
+Now start changing files in /src and refresh the browser.
+
+Files outside of /src require npm start to be rerun
+
+## Sitemap
+
+```
+/ -> /src/pages/home.js
+/encrypt/ -> /src/pages/encrypt/index.js
+/open-web-fellows/ -> /src/pages/open-web-fellows/overview.js
+/open-web-fellows/info/ -> /src/pages/open-web-fellows/info.js
+/open-web-fellows/fellows/ -> /src/pages/open-web-fellows/fellows.js
+```
+
+## Source Structure
+
+Running `npm start` is the entry point to build our code and run our server. All our npm scripts and npm dependencies are stored in package.json.
+
+Our client side code is written in react and lives in `/src`. Our files are organized into two directories, `/src/components/` and `/src/pages/`. Components are shared UI elements, like header and footer used to build pages.
+
+We have an npm script that uses [babel](https://babeljs.io/) to compile the contents of `/src` from [es6](https://en.wikipedia.org/wiki/ECMAScript#6th_Edition) to `/dist` as [es5](https://en.wikipedia.org/wiki/ECMAScript#5th_Edition). These are built to run on the server and client.
+
+We use [webpack](https://webpack.github.io/) to package our client side files into `/public`, which is handled in `webpack.config.js`
+
+We use [react-router](https://github.com/reactjs/react-router) to handle the site structure, which live in `/src/avocacy-main.js` and `/src/encrypt-main.js`.
 
-### Layout
+We use [node](http://nodejs.org/) version 4 or higher to run our server, which is handled in `app.js`.
 
-TODO
+"Encrypt" pages are handled a bit differently from the rest. While the standard routes, and source, uses `/src/advocacy-main.js` for pretty much all the generating and packaging of files, routes, and source. Encrypt uses `/src/encrypt-app.js` to package client side files, and uses `/src/encrypt-main.js` and `/src/lib/react-server-rooute.js` to handle server side rendering.
```

Comment:
`While the standard routes, and source, uses` => `The standard routes, and source, use`